edgebuilder-install.sh

```diff
@@ -500,64 +491,44 @@ uninstall_server()
 # Uninstall the Node components
 uninstall_node()
 {
-  if dpkg -s edgebuilder-node; then
-
-      # attempt autoremove
-      if sudo apt autoremove -qq edgebuilder-node -y ;then
-          echo "Successfully autoremoved node components"
+   # check if edgebuilder-node is currently installed
+   if dpkg -s edgebuilder-node; then
+      sudo edgebuilder-node down -v
```

Comment:
I dont think there's a -v option for edgebuilder-node script as it sets the -v option internally when it calls docker-compose